test/IntegrationTests/MSTest.IntegrationTests/Parameterized tests/DataExtensibilityTests.cs

```diff
@@ -3,6 +3,8 @@
 
 using System.Collections.Immutable;
 
+using FluentAssertions;
```

Comment:
If you are not adding any fluent assertion to a file, please do not add the using.